main.py

```diff
@@ -1,7 +1,50 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Actor, Genre
```

Comment:
The import statement is correct, importing the necessary models `Actor` and `Genre` from `db.models`.